### PR TITLE
Test station names and add new translations

### PR DESCRIPTION
**Pull request by mgalibert**
- I'm now testing station names to make sure they don't have special characters.
- I took the chance to add some DB ids based on Sentry Hafas errors.
- I fixed some station names here and there.
- I refined the test on station localized information
- I added hundred/thousands of translations thanks to Geonames.

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -8934,7 +8934,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 10809;Braunschweig Hbf;braunschweig-hbf;8013240;;52.252236;10.539699;;f;DE;f;Europe/Berlin;t;;;f;;f;8000049;t;;f;;f;;f;;;;f;;f;;f;;f;DEBSW;f;f;;;Main station;Brunswick Estación central;Gare centrale;Stazione centrale;;;;;ブラウンシュヴァイク;브라운슈바이크;Brunszwik;;Брауншвейг;;;不伦瑞克
 10810;Bünde (Westf);bunde-westf;8021173;;52.202076;8.573875;;f;DE;f;Europe/Berlin;t;DEAFG;;f;;f;8000059;t;;f;;f;;f;;;;f;;f;;f;;f;DEAFG;f;f;;;;;;;;;;;;;;;Бюнде;;;
 10811;Siershahn;siershahn;8019581;;50.485388;7.771561;;f;DE;f;Europe/Berlin;t;;;f;;f;8000060;t;;f;;f;;f;;;;f;;f;;f;;f;DESHN;f;f;;;;;;;;;;;;;;;;;;
-10812;Burg-u. Nieder Gemünden;burg-u-nieder-gemunden;8005610;;50.691115;9.047606;;f;DE;f;Europe/Berlin;t;;;f;;f;8000061;t;;f;;f;;f;;;;f;;f;;f;;f;DEBGE;f;f;;;;;;;;;;;;;;;;;;
+10812;Burg- und Nieder-Gemünden;burg-und-nieder-gemunden;8005610;;50.691115;9.047606;;f;DE;f;Europe/Berlin;t;;;f;;f;8000061;t;;f;;f;;f;;;;f;;f;;f;;f;DEBGE;f;f;;;;;;;;;;;;;;;;;;
```
Comment (avandecreme) [inline stations.csv]: Is the space after the dash expected? (or the dash before the space).

File: stations.csv [hunk 2]
```diff
@@ -11627,7 +11627,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 13503;Steinebach;steinebach;8020340;;48.061573;11.207229;;f;DE;f;Europe/Berlin;t;;;f;;f;8005699;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 13504;Steinefrenz;steinefrenz;8019587;;50.448415;7.938886;;f;DE;f;Europe/Berlin;t;;;f;;f;8005700;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 13505;Steinen;steinen;8014445;;47.641606;7.741564;;f;DE;f;Europe/Berlin;t;;;f;;f;8005701;t;;f;;f;;f;;;;f;;f;;f;;f;DESNN;f;f;;;;;;;;;;;;;;;;;;
-13506;Steinhagen (Westf) Bielef. Str.;steinhagen-westf-bielef-str;8062792;;52.011415;8.436457;;f;DE;f;Europe/Berlin;t;;;f;;f;8005702;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+13506;Steinhagen Bielefelder Straße (Westf) ;steinhagen-bielefelder-strasse-westf;8062792;;52.011415;8.436457;;f;DE;f;Europe/Berlin;t;;;f;;f;8005702;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```
Comment (avandecreme) [inline stations.csv]: There is an extra space after the )

File: stations.csv (hunk 1 above)
Comment (mgalibert) [inline stations.csv]: Yes, it's intended.
https://de.wikipedia.org/wiki/Bahnstrecke_Kirchhain%E2%80%93Burg-_und_Nieder-Gem%C3%BCnden

File: stations.csv (hunk 2 above)
Comment (mgalibert) [inline stations.csv]: Thanks, I'm going to forbid that as well.

**Comment by mgalibert** [comment]
Comments taken into account.

File: stations.csv [hunk 3]
```diff
@@ -10941,7 +10941,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 12817;Mammendorf;mammendorf;8002147;;48.216825;11.174616;;f;DE;f;Europe/Berlin;t;;;f;;f;8004204;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 12818;Nassenbeuren;nassenbeuren;8002228;;48.078068;10.503796;;f;DE;f;Europe/Berlin;t;;;f;;f;8004207;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 12819;Natrup-Hagen;natrup-hagen;8021023;;52.209852;7.91075;;f;DE;f;Europe/Berlin;t;;;f;;f;8004208;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-12820;Nauheim (b Gr.Gerau);nauheim-b-gr-gerau;8011234;;49.942134;8.449932;;f;DE;f;Europe/Berlin;t;;;f;;f;8004210;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```
Comment (mckbrmn) [inline stations.csv]: `Nauheim (bei Groß Gerau)`

File: stations.csv [hunk 4]
```diff
@@ -10886,12 +10886,12 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
+12765;München Leienfelsstraße;munchen-leienfelsstrasse;8020356;;48.154512;11.428336;;f;DE;f;Europe/Berlin;t;;;f;;f;8004133;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 12766;München Leuchtenbergring;munchen-leuchtenbergring;8020086;;48.133918;11.616085;;f;DE;f;Europe/Berlin;t;;;f;;f;8004134;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 12767;München Marienplatz;munchen-marienplatz;8020553;;48.137046;11.575382;;f;DE;f;Europe/Berlin;t;;;f;;f;8004135;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 12768;München Rosenheimer Platz;munchen-rosenheimer-platz;8020555;;48.129171;11.593046;;f;DE;f;Europe/Berlin;t;;;f;;f;8004136;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 12769;München Siemenswerke;munchen-siemenswerke;8020285;;48.0945;11.5328;;f;DE;f;Europe/Berlin;t;;;f;;f;8004137;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-12770;München St.Martin-Str.;munchen-st-martin-str;8020557;;48.118519;11.595689;;f;DE;f;Europe/Berlin;t;;;f;;f;8004138;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+12770;München St-Martin-Straße;munchen-st-martin-strasse;8020557;;48.118519;11.595689;;f;DE;f;Europe/Berlin;t;;;f;;f;8004138;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```
Comment (mckbrmn) [inline stations.csv]: `München St.-Martin-Straße` would be better

File: stations.csv [hunk 5]
```diff
@@ -11627,7 +11627,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 13503;Steinebach;steinebach;8020340;;48.061573;11.207229;;f;DE;f;Europe/Berlin;t;;;f;;f;8005699;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 13504;Steinefrenz;steinefrenz;8019587;;50.448415;7.938886;;f;DE;f;Europe/Berlin;t;;;f;;f;8005700;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 13505;Steinen;steinen;8014445;;47.641606;7.741564;;f;DE;f;Europe/Berlin;t;;;f;;f;8005701;t;;f;;f;;f;;;;f;;f;;f;;f;DESNN;f;f;;;;;;;;;;;;;;;;;;
-13506;Steinhagen (Westf) Bielef. Str.;steinhagen-westf-bielef-str;8062792;;52.011415;8.436457;;f;DE;f;Europe/Berlin;t;;;f;;f;8005702;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+13506;Steinhagen Bielefelder Straße (Westf);steinhagen-bielefelder-strasse-westf;8062792;;52.011415;8.436457;;f;DE;f;Europe/Berlin;t;;;f;;f;8005702;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```
Comment (mckbrmn) [inline stations.csv]: It's probably best to keep the Westfalen specifier next to the city name.

File: stations.csv (hunk 3 above)
Comment (mgalibert) [inline stations.csv]: J'ai dégagé le `bei` qui veut juste dire `à`. Ce n'est pas le nom officielle de la ville.

File: stations.csv [hunk 6]
```diff
@@ -13203,8 +13203,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 15079;Laaske;laaske;8027557;;53.202162;12.063254;;f;DE;f;Europe/Berlin;t;;;f;;f;8012144;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 15080;Lampertswalde;lampertswalde;8004279;;51.308476;13.680377;;f;DE;f;Europe/Berlin;t;;;f;;f;8012145;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 15081;Lancken;lancken;8028493;;54.51803;13.607888;;f;DE;f;Europe/Berlin;t;;;f;;f;8012146;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-15082;Landsberg (b. Halle/Saale);landsberg-b-halle-saale;8023151;;51.532775;12.166872;;f;DE;f;Europe/Berlin;t;;;f;;f;8012147;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-15083;Landsberg (b. Halle/Saale) Süd;landsberg-b-halle-saale-sud;8023394;;51.504396;12.172311;;f;DE;f;Europe/Berlin;t;;;f;;f;8012148;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+15082;Landsberg (Halle/Saale);landsberg-halle-saale;8023151;;51.532775;12.166872;;f;DE;f;Europe/Berlin;t;;;f;;f;8012147;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```
Comment (mckbrmn) [inline stations.csv]: Keep the `bei`

File: stations.csv [hunk 7]
```diff
@@ -13203,8 +13203,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
+15083;Landsberg Süd (Halle/Saale);landsberg-sud-halle-saale;8023394;;51.504396;12.172311;;f;DE;f;Europe/Berlin;t;;;f;;f;8012148;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```
Comment (mckbrmn) [inline stations.csv]: Leave the `Süd` after the city's specifier (which is missing its `bei`)

File: stations.csv (hunk 3 above)
Comment (mgalibert) [inline stations.csv]: Mounir agrees with my version.

File: stations.csv (hunk 5 above)
Comment (mgalibert) [inline stations.csv]: Germany compliant.

File: stations.csv (hunk 6 above)
Comment (mgalibert) [inline stations.csv]: Germany compliant.

**Comment by mgalibert** [comment]
Comments taken into account.
Some DB ids added + some improvements on station names.

**Comment by paulRbr** [comment]
_:warning: This comment is not related to the changes itself._

I believe we could make an effort on PR's titles and descriptions. It would be much easier for anyone following the changes made in this repository to understand what's going on. Easier for communications purposes too. E.g. when I receive an email about a PR called "Test station names" I feel it doesn't describe anything about the changes.

What do you folks think?

**Comment by mgalibert** [comment]
In this case, there is nothing particular to be understood. I'm just adding a test for station names. It's true I'm not making any effort on PR's titles and descriptions. But commits' titles are usually straight forward.

**Comment by mgalibert** [comment]
Hello all, I have updated the title and description of the PR. I took the chance this weekend to run a script and add more translations the stations file. I would love to have some to review and merge before it creates any conflict. Thanks!

**Comment by paulRbr** [comment]
@mgalibert. Thanks a lot its much more enjoyable to discover your PR now :simple_smile:. I will let @mckbrmn merge

**Pull request merged**